zellij-utils/src/cli.rs

```diff
@@ -9,6 +9,26 @@ use serde::{Deserialize, Serialize};
 use std::path::PathBuf;
 use url::Url;
 
+fn validate_session(name: &str) -> Result<String, String> {
+    #[cfg(unix)]
+    {
+        let mut socket_path = crate::consts::ZELLIJ_SOCK_DIR.clone();
+        let available_length = 108_usize
+            .saturating_sub(socket_path.as_os_str().len())
+            .saturating_sub(1);
+        socket_path.push(name);
+        if socket_path.as_os_str().len() >= 108 {
```

Comment:
Consider extracting it into some constant just because it's being reused here and under `available_length`

Reply:
I think it is good idea. @deepsghimire could you please update this? Or if you're busy can I work on this?

Reply:
Hi, yes you can work on this.